src/sources/grib.jl

```diff
@@ -0,0 +1,223 @@
+const CFG = CfGRIB
+
+const GRIB_DIMMAP = Dict(
+    "lat" => Y,
+    "latitude" => Y,
+    "lon" => X,
+    "long" => X,
+    "longitude" => X,
+    "valid_time" => Ti,
+    "time" => Ti,
+    "lev" => Z,
+    "mlev" => Z,
+    "level" => Z,
+    "vertical" => Z,
+    "x" => X,
+    "y" => Y,
+    "z" => Z,
+    "band" => Band,
+)
+
+haslayers(::Type{GRIBfile}) = true
+defaultcrs(::Type{GRIBfile}) = EPSG(4326)
```

Comment:
I admit I just copied from `ncdasasets.jl` without really understanding, just hoping to break as less things as possible :relaxed: I don't really about CRS with GRIB

Reply:
Hah yeah there are corners of all the raster data specs I don't care about either.

But if you don't implement it correctly someone will post an issue here one day that forces me to fix it instead...

Reply:
Sorry for the typo, I didn't mean that I don't care, but that I don't know about CRS with GRIB :wink: 
I did some research and it happens that knowing the CRS from the GRIB messages is not straightforward. Reading the shapeOfTheEarth key and connecting it to [this table](https://apps.ecmwf.int/codes/grib/format/grib2/ctables/3/2) can give information about the reference system. Then I guess it could be translated to standard CRS (with PROJ4?), but I'm really not familiar with all that. I guess that for the moment we could do `defaultcrs(::Type{GRIBfile}) = nothing` and leave it to the user to determine what CRS he's dealing with?